cdm/src/main/java/thredds/client/catalog/Catalog.java

```diff
@@ -136,6 +142,18 @@ private Service findService(List<Service> services, String want)  {
     return null;
   }
 
+  private Service findService(List<Service> services, ServiceType type)  {
```

Comment:
I only ask because I know we had quite a few Coverity issues like this. I don't have a stake in it, other than to prevent the Coverity # going up.